spec/requests/admin/users_controller_spec.rb

```diff
@@ -1477,18 +1479,87 @@
         expect(response.status).to eq(200)
         expect(User.where(id: deleted_users.map(&:id)).count).to eq(0)
       end
+
+      it "blocks emails and IPs of deleted users if block_ip_and_email is true" do
+        current_user.update!(ip_address: IPAddr.new("127.189.34.11"))
+        deleted_users[0].update!(ip_address: IPAddr.new("127.189.34.11"))
+        deleted_users[1].update!(ip_address: IPAddr.new("249.21.44.3"))
+        deleted_users[2].update!(ip_address: IPAddr.new("3.1.22.88"))
+
+        expect do
+          delete "/admin/users/destroy-bulk.json",
+                 params: {
+                   user_ids: deleted_users.map(&:id),
+                   block_ip_and_email: true,
+                 }
+        end.to change {
+          ScreenedIpAddress.where(action_type: ScreenedIpAddress.actions[:block]).count
+        }.by(2).and change {
+                ScreenedEmail.where(action_type: ScreenedEmail.actions[:block]).count
+              }.by(3)
+
+        expect(
+          ScreenedIpAddress.where(
```

Comment:
All these `.where...to exist` calls can be changed to `expect(ScreenedIpAddress.exists?()).to be_truthy`